app/Console/Commands/FetchCollectionOpenseaSlug.php

```diff
@@ -30,6 +30,8 @@ class FetchCollectionOpenseaSlug extends Command
      */
     public function handle(): int
     {
+        $limit = $this->option('limit');
```

Comment:
I'd approach this differently. Instead of calling this command every 5 seconds and determining which collections have been updated since (and requiring a query for it), what about:
- keep the scheduled task on an hourly basis
- query one time for all collections that need updating
- then chunk those collections based on the allowed amounts for the ratelimit (e.g. 3)
- for each chunk, delay it by x seconds so they run after one another

That should handle the scenario in a similar way, but without the need to constantly query collections (every 5 seconds) when most collections won't require an update for the majority of the time. There are some caveats with this approach but they apply to the existing one as well, e.g. how we would handle failed jobs that get retried and clash with the newly scheduled ones. For this we should consider removing retries completely for certain jobs and let them fail and be re-scheduled later to not create additional jobs. Maybe we can already remove the retries for this job as it will be included in the next time the schedule gets triggered

@alfonsobries @oXtxNt9U